parser_library/test/workspace/pathmask_test.cpp

```diff
@@ -92,10 +92,10 @@ TEST(pathmask, pass)
     EXPECT_TRUE(check_path("/path/?/test/", "/path/%EF%BF%BF/test/"));
     EXPECT_TRUE(check_path("/path/?/test/", "/path/%F0%9F%A7%BF/test/"));
 
-    EXPECT_TRUE(check_path("/path/??/test/", "/path/%7D%DF%BF/test/"));
-    EXPECT_TRUE(check_path("/path/??/test/", "/path/%DF%BF%7D/test/"));
-    EXPECT_TRUE(check_path("/path/??/test/", "/path/%EF%BF%BF%7D/test/"));
-    EXPECT_TRUE(check_path("/path/???/test/", "/path/%F0%9F%A7%BF%7D%DF%BF/test/"));
+    EXPECT_TRUE(check_path("/path/\?\?/test/", "/path/%7D%DF%BF/test/"));
```

Comment:
aren't trigraphs supposed to be banned in like c++17?

Reply:
Well, Clang issued warnings about it so it was either this or suppressing the warning.

Reply:
well, that's unfortunate :(